server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds.Contracts/Interfaces/IHelpSettings.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Arcadia.Assistant.AppCenterBuilds.Contracts
```

Comment:
R# settings should have complained on namespace not being equal to a folder. Do you really need interfaces folder? Interfaces and Classes folders (as well as Impl postfix) is just a noise that doesn't give any context. User should not know whether he operates with interface or a class

Reply:
Done